library/src/main/java/com/novoda/downloadmanager/DownloadBatch.java

```diff
@@ -289,9 +292,12 @@ private static boolean networkError(InternalDownloadBatchStatus downloadBatchSta
     }
 
     void pause() {
-        Logger.v("pause batch " + downloadBatchStatus.getDownloadBatchId().rawId() + ", " + STATUS + " " + downloadBatchStatus.status());
+        Logger.v("pause " + BATCH + downloadBatchStatus.getDownloadBatchId().rawId() + STATUS + downloadBatchStatus.status());
         DownloadBatchStatus.Status status = downloadBatchStatus.status();
```

Comment:
I'm commenting it here, but this is valid for most of these methods:
if you move `downloadBatchStatus.status()` at the beginning of the method, then you can directly use `status` in all the logs and checks

Reply:
ea465c4